src/Utils/QueryAnalyzer.php

```diff
@@ -483,46 +512,42 @@ public function track_nodes( $model ) {
 		return $model;
 	}
 
-
 	/**
-	 * Return headers
-	 *
-	 * @param array $headers The array of headers being returned
+	 * Returns graphql keys for use in debugging and headers.
 	 *
 	 * @return array
 	 */
-	public function get_headers( array $headers = [] ): array {
+	public function get_graphql_keys() {
 
-		$keys = [];
+		if ( ! empty( $this->graphql_keys ) ) {
+			return $this->graphql_keys;
+		}
+
+		$keys        = [];
+		$return_keys = '';
 
 		if ( $this->get_query_id() ) {
 			$keys[] = $this->get_query_id();
 		}
 
 		if ( ! empty( $this->get_root_operation() ) ) {
-			$headers['X-GraphQL-Operation-Type'] = $this->get_root_operation();
-			$keys[]                              = 'graphql:' . $this->get_root_operation();
+			$keys[] = 'graphql:' . $this->get_root_operation();
 		}
 
 		if ( ! empty( $this->get_list_types() ) && is_array( $this->get_list_types() ) ) {
-			$headers['X-GraphQL-List-Types'] = implode( ' ', array_unique( array_values( $this->get_list_types() ) ) );
```

Comment:
no longer setting this header. intentional?

Reply:
Ya, similar to `X-GraphQL-Nodes`, this data is already grouped in the `X-GraphQL-Keys` and with the limits, I don't want to output redundant data.